androidHackathonApp/src/main/res/layout/activity_main.xml

```diff
@@ -9,13 +9,42 @@
     tools:context=".MainActivity">
 
   <TextView
-      android:layout_width="wrap_content"
+      android:id="@+id/textView"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:text="Hello World!"
-      android:id="@+id/text_view"
+      android:layout_marginBottom="32dp"
+      android:layout_marginLeft="24dp"
+      android:layout_marginRight="24dp"
+      android:text="@string/subheading"
+      app:layout_constraintBottom_toTopOf="@+id/add_meal"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintStart_toStartOf="parent" />
+
+  <com.kcc.kmmhackathon.androidHackathonApp.ActionButton
+      android:id="@+id/add_meal"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginLeft="24dp"
+      android:layout_marginRight="24dp"
+      android:text="@string/plusButton"
+      android:background="@color/colorPrimary"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintLeft_toLeftOf="parent"
-      app:layout_constraintRight_toRightOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+  <com.kcc.kmmhackathon.androidHackathonApp.ActionButton
+      android:id="@+id/find_meal"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="32dp"
+      android:layout_marginLeft="24dp"
+      android:layout_marginRight="24dp"
+      android:text="@string/findButton"
+      android:background="@color/colorSecondary"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/add_meal" />
 
```

Comment:
Could have the style for these buttons as one custom button then just add the text/colour for each? They could also be in their own vertical LinearLayout to save having to constraint them both? Just some (maybe wrong) suggestions to try shorten this :)